src/sparseml/__init__.py

```diff
@@ -53,5 +53,8 @@
     )
 except Exception as err:
     print(
-        f"Need sparsezoo version above 0.9.0 to run Neural Magic's latest-version check\n{err}"
+        f"Need sparsezoo version above 0.9.0 to run Neural Magic's latest-version "
+        f"check\n{err}"
```

Comment:
your ide formatter looks like it might be a bit different from our standardized black formatter, specifically on character limit here - can you look into either setting up, or only using black for formatting to avoid unintended changes like this?

Reply:
The line was more than 88 characters so I wrapped it, PEP 8 defines a 80 character limit, is using two lines bad?